packages/cloud_firestore/cloud_firestore/ios/cloud_firestore.podspec

```
@@ -24,17 +24,18 @@ Pod::Spec.new do |s|
   s.license          = { :file => '../LICENSE' }
   s.authors          = 'The Chromium Authors'
   s.source           = { :path => '.' }
-  
+
   s.source_files     = 'Classes/**/*.{h,m}'
   s.public_header_files = 'Classes/Public/*.h'
   s.private_header_files = 'Classes/Private/*.h'
 
-  s.ios.deployment_target = '10.0'
+  s.ios.deployment_target = '11.0'
   s.dependency 'Flutter'
 
   s.dependency 'firebase_core'
   s.dependency 'Firebase/Firestore', firebase_sdk_version
-  
+  s.dependency 'nanopb'
```

I logged an issue with the appropriate details, and the dependency here should have the same version constraint as upstream I think?

```suggestion
  # required until firestore-ios-sdk-frameworks is updated, otherwise users of that distribution will have compile failures
  # see https://github.com/invertase/firestore-ios-sdk-frameworks/issues/59
  s.dependency 'nanopb', '>= 2.30908.0', '< 2.30910.0'
```